app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -197,7 +206,7 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
             if (p1 != null) {
                 if (p1.action.equals(WifiManager.SCAN_RESULTS_AVAILABLE_ACTION)) {
                     val wifiList = mainWifi.getScanResults()
-                    Timber.d("Check%s",checkDevice)
+                    Timber.d("Check%s", checkDevice)
```

Comment:
Not much imp but space can be added after Check i.e. "Check %s"

Reply:
I used default android studio styling so I don't know if this matters, plus it is a debug string so not subjected for review

Reply:
This is not default styling. It depends on whether you add space or not. I already said its not imp but if you feel like doing this you may while making other changes. Just mentioned. Afterall many people work on the project together, so there's nothing wrong in keeping the messages/strings proper.